lib/ui/views/about/about_tos_view.dart

```diff
@@ -0,0 +1,235 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AboutTosView extends StatelessWidget {
+  static const String id = 'about_tos_view';
+
+  final bool showAppBar;
+
+  const AboutTosView({Key key, this.showAppBar = true}) : super(key: key);
+
+  TextSpan _buildText(BuildContext context, String text, {bool bold = false}) {
+    return TextSpan(
+        style: Theme.of(context)
+            .textTheme
+            .bodyText1
+            .copyWith(fontWeight: bold ? FontWeight.bold : FontWeight.normal),
+        text: text);

```

Comment:
Adding a comma after text will have a better indentation and goes well with other parts of the application.

Reply:
Done